### PR TITLE
Кирьяк Александр

### PR DESCRIPTION
**Pull request by kryag**
[codesandbox](https://codesandbox.io/p/sandbox/zealous-jennings-kqkr3l?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm0zx9tby0006356i8661b094%2522%252C%2522sizes%2522%253A%255B100%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm0zx9tbx0002356iyxbx0b7v%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm0zx9tbx0003356i32g0fioj%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm0zx9tbx0005356i8hcw4okl%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm0zx9tbx0002356iyxbx0b7v%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0zx9tbx0001356idy6n81nh%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.html%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522cm0zxbfif001x356ia1l2qmmo%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fstyles.css%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522cm0zx9tbx0002356iyxbx0b7v%2522%252C%2522activeTabId%2522%253A%2522cm0zxbfif001x356ia1l2qmmo%2522%257D%252C%2522cm0zx9tbx0005356i8hcw4okl%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0zx9tbx0004356i5871kmzs%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%257D%255D%252C%2522id%2522%253A%2522cm0zx9tbx0005356i8hcw4okl%2522%252C%2522activeTabId%2522%253A%2522cm0zx9tbx0004356i5871kmzs%2522%257D%252C%2522cm0zx9tbx0003356i32g0fioj%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522cm0zx9tbx0003356i32g0fioj%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="description" content="Autobiography">
+    <meta name="author" content="Aleksandr Kiriak">
```
Comment (mokhov) [inline index.html]: Почему при "ru" в "lang" пишешь на английском?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+<main>
+    <article>
+        <header>
+            <h1 style="text-align: center;">Автобиография</h1>
+        </header>
+
+        <figure style="float: right;">
```
Comment (mokhov) [inline index.html]: За старание хвалю, но в целом если поизменять окно браузера, то увидишь все недостатки. Без CSS но для разметки тут было бы проще всё в таблицу запихать

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            <img src="resources/my_photo.jpg" alt="Моё фото" width="500">
+            <figcaption style="text-align: center">Это я</figcaption>
+        </figure>
+
+        <p>
+            Меня зовут Саша, мне 20 лет. Я учусь на третьем курсе в <em>Университете ИТМО</em> на направлении
```
Comment (mokhov) [inline index.html]: я бы «Университете ИТМО» сделал бы ссылкой и ИТМО бы ещё допом в abbr упаковал

File: index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            "Прикладная математика и информатика". Параллельно прохожу стажировку в <strong>Т-Банке</strong>.
```
Comment (mokhov) [inline index.html]: И тут тоже

File: index.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            <h3>Пару фактов обо мне:</h3>
+            <ol type="I">
+                <li>Кандидат в мастера спорта по плаванию.</li>
+                <li>Любимый фильм: Начало (<time datetime="2010">2010</time>).</li>
+                <li>Любимая цитата: <q>Своим долголетием я обязан спорту. Я им никогда не занимался</q> — Уинстон Черчилль.</li>
+            </ol>
+        </section>
+
+        <p>В процессе работы и учёбы в университете я получил следующие знания:</p>
+        <div style="font-family: Georgia, serif; text-align: center;">
```
Comment (mokhov) [inline index.html]: А зачем тут div? он кажется лишний

File: index.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+            <table>
+                <tbody>
+                <tr>
+                    <td>Языки программирования</td>
```
Comment (mokhov) [inline index.html]: Здесь правильнее использовать th

File: index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                </tr>
+                </tbody>
+            </table>
+        </div>
+        <p>
+            Хочу пополнить эту таблицу знаниями <mark style="background-color: greenyellow;">Фронтенда</mark>,
+            собственно поэтому я и пишу это письмо.
+        </p>
+
+        <video width="350" height="280" poster="resources/4etko_preview.jpg" controls loop>
```
Comment (mokhov) [inline index.html]: :)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+            <source src="resources/4etko.ogv" type="video/ogg">
+            Элемент video не поддерживается вашим браузером.
+            <a href="resources/4etko.mp4">Скачайте видео</a>
+        </video>
+    </article>
+</main>
+
+<hr>
+<pre>
+────▄████▄▄
```
Comment (mokhov) [inline index.html:77]: 👍

**Review by mokhov — COMMENTED** [review]
В целом ОК, можно чуть-чуть лучше

**Comment by kryag** [comment]
Исправил замечания, ссылка на codesandbox не изменилась

**Comment by mokhov** [comment]
Принято

**Pull request closed without merging**